style.css

```diff
@@ -403,3 +415,135 @@ textarea::placeholder {
   font-size: 15px;
   padding: 10px;
 }
+
+@media (min-width: 768px) {
+  body {
+    font-family: "poppins", sans-serif;
+    background: #f7f7f9;
+    padding: 0;
+    margin: 0;
+    width: 100%;
+  }
+
+  header {
+    padding-left: 10%;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .items {
+    display: flex;
+  }
+
+  nav > i {
+    display: none;
+  }
+
+  .main-section {
+    display: flex;
+    flex-direction: column;
+    width: inherit;
+  }
+
+  .headline {
+    height: 100vh;
+    background-image: url(assets/headerbg.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+    border-bottom-left-radius: 160px;
```

Comment:
DESKTOP VERSION -

- [ ] Kindly fix the spacing as per shown in videos 1 and 2, following the Figma design. For the intro section and the about section.
- [ ] Kindly make sure that your card image's height is as per the Figma design. When your design is increased images are getting smaller, kindly fix it.
- [ ] Kindly make sure your card displays an image in 50% of its width and text takes 50% of the card.
- [ ] Kindly follow the spacing for the about section and fix the margin for it when the desktop size increases. Your design at 768px is perfect and when the size increases from 768, the elements should display in the same way following the spacing.
- [ ] Kindly follow the same card images as per the Figma design.

Video 1:

https://user-images.githubusercontent.com/70269507/171370081-173f8b71-a3c4-4432-acaa-b916cb37a779.mov

Video 2:

https://user-images.githubusercontent.com/70269507/171370032-874d9547-19a9-46d8-9f7e-5edab6f9ab74.mov